piker/data/_ahab.py

```diff
@@ -133,6 +133,136 @@ def unpack_msg(err: Exception) -> str:
                 c.kill()
 
 
+class Container:
```

Comment:
What you think of creating a `docker-utils` python package with a lot of common abstractions to deal with dockerisms, it would be diferent than `pytest-dockerctl` cause it wouldnt be only about pytest, I have a bunch of utils id like in a pip instalable package somewhere

Reply:
Examples: https://github.com/guilledk/py-eosio/blob/71065aad8a792a54b05f63b8ce8b4ac68b00b1e0/py_eosio/sugar.py#L338

`wait_for_attr`, `get_container`, `docker_open_process`, `docker_wait_process`, `docker_move_into`

Reply:
i think this is an amazing idea!

for example here, all the supervisor stuff i had to hack would be great to get into a (somewhat) formalized api layer.

Reply:
Yeah I'm also not settled on a lot of this api / naming so don't take it too too seriously and feel free to suggest changes.

I'm also fine with landing this code and making it possible for someone else to go through and do a cleanup / reorg if desired.

I mostly just want to get in the ability to store large data sets for users and prime things for proper real-time ingest from tick feeds.